install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -581,6 +581,12 @@ ingressController:
     labels: {}
     # -- Annotations to be added for the shared LB service
     annotations: {}
+    # -- Service type for the shared LB service
+    type: LoadBalancer
+    # -- Configure a specific nodePort for insecure HTTP traffic on the shared LB service
+    insecureNodePort: ~
+    # -- Configure a specific nodePort for secure HTTPS traffic on the shared LB service
+    secureNodePort : ~
```

Comment:
Do these mean that, for the NodePort settings to work, the `type` should be `NodePort`? Do we normally record that in this part of the docs, or somewhere else?

Reply:
Yea, type needs to be nodePort for you to set explicit nodePorts. K8s validates this iirc, so we shouldn’t need to. Setting the nodePorts explicitly also isn’t _required_